configure.py

```diff
@@ -382,13 +384,12 @@ def set_build_var(environ_cp, var_name, query_item, option_name,
 
   var = str(int(get_var(environ_cp, var_name, query_item, enabled_by_default)))
   environ_cp[var_name] = var
-  if var == '1':
-    write_to_bazelrc('build --define %s=true' % option_name)
-  elif bazel_config_name is not None:
-    # TODO(mikecase): Migrate all users of configure.py to use --config Bazel
-    # options and not to set build configs through environment variables.
-    write_to_bazelrc('build:%s --define %s=true'
-                     % (bazel_config_name, option_name))
+  # TODO(mikecase): Migrate all users of configure.py to use --config Bazel
+  # options and not to set build configs through environment variables.
+  if var=='1':
```

Comment:
Several changes in this file don't follow [TF coding style](https://www.tensorflow.org/community/style_guide).

Reply:
Please don't change this file at the moment, our CL for the configuration will contain the fix.

Reply:
@aaroey just to be clear, should we remove configure.py from the PR?

Reply:
@samikama Yes, please exclude this file at the moment.